internal/graph/check.go

```diff
@@ -230,8 +226,7 @@ func union(ctx context.Context, concurrencyLimit uint32, handlers ...CheckHandle
 	return &ResolveCheckResponse{
 		Allowed: false,
 		ResolutionMetadata: &ResolveCheckResponseMetadata{
```

Comment:
i want to send another PR where `ResolveCheckResponseMetadata` is not a pointer.

Reply:
https://github.com/openfga/openfga/pull/2061